### PR TITLE
Move dependabot config to .github folder

### PR DESCRIPTION
**Pull request by SVilgelm**
Replace existent dependabot config with v2 and move it to the `.github` folder
Enable version updates for GitHub actions

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: 2
```
Comment (sayboras) [inline .github/dependabot.yml:1]: awesome

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+updates:
+  - package-ecosystem: gomod
```
Comment (sayboras) [inline .github/dependabot.yml:3]: nit: you can add _assignees_ so that new PR will notify the team if you want.

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    directory: "/"
+    schedule:
+      interval: weekly
+    ignore:
+      # ignore forked linters because of their versioning issues. TODO: fix it.
+      - dependency-name: "github.com/golangci/*"
+  - package-ecosystem: github-actions
```
Comment (sayboras) [inline .github/dependabot.yml:10]: so great to have this one to update github-actions as well.

I just take a look at the all supported package-ecosystem. Docker might be another one we can enable.

File: .github/dependabot.yml (hunk 2 above)
Comment (SVilgelm) [reply .github/dependabot.yml:3]: probably reviewers

File: .github/dependabot.yml (hunk 3 above)
Comment (SVilgelm) [reply .github/dependabot.yml:10]: let me check it

Comment (SVilgelm) [reply .github/dependabot.yml:12]: done

**Pull request merged**